schwab/orders/options.py

```diff
@@ -89,32 +87,45 @@ def __init__(self, underlying_symbol, expiration_date, contract_type,
                 'Strike price must be a string representing a positive ' +
                 'float')
 
-        # Remove extraneous zeroes at the end
-        strike_copy = strike_price_as_string
-        while strike_copy[-1] == '0':
-            strike_copy = strike_copy[:-1]
-        if strike_copy[-1] == '.':
-            strike_price_as_string = strike_copy[:-1]
+        # convert strike to Schwab's format
+        # Unit Test Examples (4 variations): 
+        # QQQ  240424P00500000 (Apr 24, 2024 Put Strike 500)
+        # SPXW  240422C05040000 (Apr 22, 2024 Call Strike 5040)
+        # QQQ   240621P00404780 (Jun 21, 2024 Put Strike 404.78)
+        # SPXW  240422C05040500 (Apr 22, 2024 Call Strike 5040.50)
+        
+        if strike.is_integer() and strike < 1000:
+            self.strike_price = '00' + strike * 1000
```

Comment:
these are going to fail with `TypeError: can only concatenate str (not "int") to str`
suggest using something like `strike_price =  str(strike * 1000).rjust(8, '0')`

Reply:
Fixed